content/02.main-text.md

```diff
@@ -86,7 +83,14 @@ In addition, git carefully tracks all revisions to the manuscript text but not t
 These discussions must be archived to ensure that important decisions about the manuscript are preserved and authors receive credit for intellectual contributions that are not directly reflected in the manuscript's text.
 GitHub supports programmatic access to issues, pull requests, and reviews so tracking these conversations is feasible in the future.
 
-`TODO: cgreene paragraph on diversity, open science culture, possibly author disagreements and conduct`
+It is important to consider how the move to an open contribution model affects under-represented minority members of the scientific community [@url:https://lgatto.github.io/open-and-open/].
```

Comment:
I'm not sure how the contributor guidelines discussed below relate to making the open contribution model equitable for URMs. In what way did the deep review consider this?

**Update**, I hadn't finished reading this paragraph:

> Our contributor guidelines did not discuss broader community norms that may have improved inclusiveness

Would it make sense to put all this content together? For example, start with what our contributing guidelines did address. Then say what we recommend future guidelines also address.

Reply:
👍 see if you like the revision